waypoints:
  wp0:
    position:
      x: 0.88
      y: 3.4
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.707
      w: 0.707
  wp_table_1:
    position:
      x: 0.88
      y: 3.4
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.707
      w: 0.707
  wp_shelf_1:
    position:
      x: -2.9
      y: 0.5
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 1.0
      w: 0.0
